src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -20,6 +20,8 @@
 PRODUCT_NAME = 'azurecli'
 TELEMETRY_VERSION = '0.0.1.4'
 AZURE_CLI_PREFIX = 'Context.Default.AzureCLI.'
+EXTENSION_EVENT_NAME = PRODUCT_NAME + '/extension/{}'
```

Comment:
New development: per recent discussion with Jason, the event name can't be expandable. Therefore, limit it to `azurecli/extension`.